.github/workflows/playground.yaml

```diff
@@ -0,0 +1,50 @@
+name: playground
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository_owner	}}/playground
```

Comment:
@MichaelClifford keep this model-server used in AI Lab playground as `playground`? and for other uses we'll name the model_server/llamacpp/base/Containerfile image `model-server-llamacpp` or something similar, sg?